src/Tribe/App_Shop.php

```diff
@@ -123,32 +123,166 @@ public function is_current_page() {
 		public function do_menu_page() {
 			$main = Tribe__Main::instance();
 			$products = $this->get_all_products();
+			$bundles = $this->get_bundles();
+			$extensions = $this->get_extensions();
 			include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/app-shop.php';
 		}
 
 		/**
-		 * Get's all products from the API
+		 * Gets all products from the API
 		 *
 		 * @return array|WP_Error
 		 */
 		private function get_all_products() {
 			$all_products = tribe( 'plugins.api' )->get_products();
 
-			$products = array(
-				(object) $all_products['event-aggregator'],
-				(object) $all_products['events-calendar-pro'],
-				(object) $all_products['event-tickets-plus'],
-				(object) $all_products['promoter'],
-				(object) $all_products['tribe-filterbar'],
-				(object) $all_products['events-community'],
-				(object) $all_products['events-community-tickets'],
-				(object) $all_products['tribe-eventbrite'],
-				(object) $all_products['image-widget-plus'],
-			);
+			$products = [
+				'the-events-calendar' =>      (object) $all_products['the-events-calendar'],
+				'events-calendar-pro' =>      (object) $all_products['events-calendar-pro'],
+				'virtual-events' =>           (object) $all_products['virtual-events'],
+				'event-aggregator' =>         (object) $all_products['event-aggregator'],
+				'event-tickets' =>            (object) $all_products['event-tickets'],
+				'event-tickets-plus' =>       (object) $all_products['event-tickets-plus'],
+				'promoter' =>                 (object) $all_products['promoter'],
+				'tribe-filterbar' =>          (object) $all_products['tribe-filterbar'],
+				'events-community' =>         (object) $all_products['events-community'],
+				'events-community-tickets' => (object) $all_products['events-community-tickets'],
+				'tribe-eventbrite' =>         (object) $all_products['tribe-eventbrite'],
+				'image-widget-plus' =>        (object) $all_products['image-widget-plus'],
+			];
```

Comment:
```suggestion
			$products = array_map( static function ( $value ) {
				return (object) $value;
			 } );
```

Reply:
nope. we want to specifically control the order these are displayed in.

Reply:
Ahhh I see!